packages/flutter/lib/src/material/flexible_space_bar.dart

```diff
@@ -166,11 +172,14 @@ class _FlexibleSpaceBarState extends State<FlexibleSpaceBar> {
         final Matrix4 scaleTransform = new Matrix4.identity()
           ..scale(scaleValue, scaleValue, 1.0);
         final Alignment titleAlignment = _getTitleAlignment(effectiveCenterTitle);
-        children.add(new Container(
-          padding: new EdgeInsetsDirectional.only(
-            start: effectiveCenterTitle ? 0.0 : 72.0,
+        final EdgeInsetsGeometry titlePadding = widget.padding ??
+          new EdgeInsetsDirectional.only(
+            start: 72.0,
+            end: effectiveCenterTitle ? 72.0 : 0.0,
```

Comment:
What does the "72.0" represent?  I know you didn't add this constant, but If it's really always the right answer, then it should probably be a named constant so that the name is self-documenting. (same with the 16.0 below).